src/app/components/AppBar/index.tsx

```diff
@@ -57,7 +60,7 @@ const ListItem = styled.li`
 
 const activeClassName = 'ACTIVE';
 
-const StyledNavLink = styled(NavLink).attrs({ activeClassName })`
+const StyledNavLink = styled(NavLink).attrs({ activeClassName })<{ hasNotification?: boolean }>`
```

Comment:
don't change `StyledNavLink`.
please make one specific component derived from `StyledNavLink` for the vote button.

Reply:
done